guides/common/modules/con_unsupported-use-cases-for-foreman-on-aws.adoc

```diff
@@ -1,12 +1,13 @@
-= Use Cases that Do Not Work
+[id="unsupported-use-cases-for-{project-context}-on-aws"]
+= Unsupported use cases for {Project} on AWS
 
 In AWS, you cannot manage the DHCP.
```

Comment:
Note that if you add a Smart Proxy using a Smart Proxy in our on-premise data center (see Scenario 2: Connecting on-premises and AWS region) then you can use these methods.

This is a long standing issue I have with this guide.